html/webappapis/dynamic-markup-insertion/opening-the-input-stream/reload.window.js

```diff
@@ -0,0 +1,53 @@
+// This test tests for the nonexistence of a reload override buffer, which is
+// used in a previous version of the HTML Standard to make reloads of a
+// document.open()'d document load the written-to document rather than doing an
+// actual reload of the URL.
+//
+// This test has a somewhat interesting structure compared to the other tests
+// in this directory. It eschews the <iframe> structure used by other tests,
+// since when the child frame is reloaded it should adopt the URL of the test
+// page (the responsible document of the entry settings object), and the spec
+// forbids navigation in nested browsing contexts to the same URL as their
+// parent. To work around that, we use window.open() which does not suffer from
+// that restriction.
+
+if (!opener) {
+  async_test(t => {
+    // 1. Open an auxiliary window.
+    const win = window.open("/common/blank.html");
+    t.add_cleanup(() => { win.close(); });
+
+    win.addEventListener("load", t.step_func(() => {
+      t.step_timeout(() => {
```

Comment:
What's the setTimeout for? Needs a comment.

Reply:
Good question, I think it might not be necessary anymore.

Reply:
It seems to be necessary for Firefox. Added a comment.